Planes/MilitaryPlane.js

```diff
@@ -1,16 +1,14 @@
 const Plane = require('./Plane');
 
 class MilitaryPlane extends Plane {
-
-    constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, militaryType) {
-        super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
-        this.militaryType = militaryType;
-
-    }
-
-    getMilitaryType() {
-        return this.militaryType;
-    }
+  constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, militaryType) {
```

Comment:
Pass an object if more than two parameters are required. Check all constructors and change parameters.